packages/android_intent_plus/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 3.2.0
+
+- Add `action_apn_settings` intent support
+
```

Comment:
Please revert this change as per the updated contributor guidelines (no need to update changelog or pubspec anymore)